apps/client/src/components/common/Header/index.tsx

```diff
@@ -1,13 +1,13 @@
 import { Box, Button, Text } from '@chakra-ui/react';
+import { useAtom } from 'jotai';
 import { useRouter } from 'next/navigation';
-import { useSetRecoilState } from 'recoil';
 import { LOCAL_STORAGE_KEY } from '@/constants/storage';
 import { isLoginState, useUserInformation } from '@/store/UserInformation';
 
 const Header = () => {
   const router = useRouter();
 
-  const setIsLogin = useSetRecoilState(isLoginState);
+  const [, setIsLogin] = useAtom(isLoginState);
```

Comment:
```suggestion
  const setIsLogin = useSetAtom(isLoginState);
```

useSetAtom을 사용한다면 불필요한 구조 분해 할당을 하지 않고 필요한 요소만 사용이 가능 할 것 같아여

Reply:
useAtom 사용하면서도 이상함을 느꼈었는데 그런게 있었군요!